B/basiclu/bundled/patches/cc.patch

```diff
@@ -0,0 +1,20 @@
+--- a/basiclu/config.mk
```

Comment:
Do you have to patch `config.mk`?  Can't you do
```
make CC99="cc -std=c99" LDLIBS="-lm -lrt"
```
?

Reply:
I don't need to patch `config.mk`, I can do 
```
make CC99="cc -std=c99" LDLIBS="-lm -lrt"
```
directly. I just forgot that I can do it. I will remove the patch.